lib/uploadex.ex

```diff
@@ -17,6 +17,7 @@ defmodule Uploadex do
 
       ## Files
       def store_files(record), do: Files.store_files(record, __MODULE__)
+      def store_files(record, previous_record), do: Files.store_files(record, __MODULE__, previous_record)
```

Comment:
all functions that receive `__MODULE__` do so in the last argument, can we keep this pattern in this function as well?